_posts/2020-09-26-bring-your-own-datatypes.md

```diff
@@ -227,15 +229,15 @@ This can be any function
 In our case,
   we use a helper function
   provided by the Bring Your Own Datatypes framework.
-`tvm.datatype.create_lower_func('BFloat16Add')`
+`tvm.target.datatype.create_lower_func({16:'Posit16es2Add'})`
   creates a lowering function
   for the common pattern described above.
 The resulting function
   converts the arguments of the given node
   to `uint16_t`,
   and then converts the node itself
   into a call to the given function name
-  (in this case, `'BFloat16Add'`).
+  (in this case, `'Posit16es2Add'`) for `posit`s of bit length 16.
 
```

Comment:
Can you add another sentence in here explaining the dictionary passed in to `create_lower_func`? You hint at its purpose in the last sentence, but I'd like to be more explicit

Reply:
We pass a dictionary to `create_lower_func` so that TVM can dispatch
  to the appropriate function name based on the bit length of the datatype.

do you think this is good? I don't want to be too specific since the developer tutorials contain this information and I see the blog post as more of a introduction to BYODT

Reply:
Good point on not being too specific. Yeah, that sounds good to me!